tests/test_call_uvec_multistage.py

```diff
@@ -17,37 +17,19 @@ def test_call_uvec_multi_stage():
 
     project_parameters = ["ProjectParameters_stage1.json", "ProjectParameters_stage2.json"]
 
-    cwd = os.getcwd()
-
-    # initialize model
-    model = Kratos.Model()
-
-    # loop over all stages
-    for file_name in project_parameters:
-
-        # change working directory to test file directory
-        os.chdir(test_file_dir)
-
-        # read parameters
-        with open(file_name, 'r') as parameter_file:
-            parameters = Kratos.Parameters(parameter_file.read())
-
-        # run stage
-        stage = analysis.StemGeoMechanicsAnalysis(model, parameters)
-        stage.Run()
-
-        # change working directory back to original working directory
-        os.chdir(cwd)
+    # run the analysis
+    Utils.run_stages(test_file_dir, project_parameters)
 
     # calculated disp below first wheel
     calculated_disp_file = Path(r"tests/test_data/input_data_multi_stage_uvec/output/calculated_disp")
     expected_disp_file = Path(r"tests/test_data/input_data_multi_stage_uvec/_output/expected_disp")
 
     # check if calculated disp below first wheel is equal to expected disp
-    are_files_equal = assert_floats_in_files_almost_equal(calculated_disp_file, expected_disp_file)
+    are_files_equal, message = assert_floats_in_files_almost_equal(calculated_disp_file, expected_disp_file)
+
     # remove calculated disp file as data is appended
     calculated_disp_file.unlink()
-    assert are_files_equal
```

Comment:
line 42 use os.path.join or Path insteal of /

btw why are you moving the files around?